docs/docs/classic-ml/tracking/tracking-api/index.mdx

```diff
@@ -0,0 +1,427 @@
+import Link from "@docusaurus/Link";
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+import { APILink } from "@site/src/components/APILink";
+
+# MLflow Tracking APIs
+
+[MLflow Tracking](/ml/tracking) provides Python, R, Java, or REST API to log your experiment data and models.
+
+## [Auto Logging](/ml/tracking/autolog)
+
+[Auto logging](/ml/tracking/autolog) is a powerful feature that allows you to log metrics, parameters, and models without the need for explicit log statements
+but just a single `mlflow.autolog()` call at the top of your ML code.
+
+```python
+import mlflow
+
+mlflow.autolog()
+
+# Your training code...
+```
+
+See [Automatic Logging with MLflow Tracking](/ml/tracking/autolog) documentation for supported libraries and how to use autolog APIs with each of them.
```

Comment:
Do we need three places to link to the same `/ml/tracking/autolog` page in this section? 😅